onboarding.md

```diff
@@ -2,10 +2,13 @@
 
 ## Welcome to Niteo
 
-Congratulations on completing the interview process, and joining us here at Niteo. We are very much looking forward to having you on the team! This page is designed to help you navigate the onboarding process from reading the Handbook to completing the so-called "onboarding issue". Regardless of your role, we all use GitHub for tracking what we do, and if you come from a non-technical background, this might be new to you! The onboarding issue is a single issue that tracks everything that needs to be done for a successful onboarding. At times the onboarding issue might feel overwhelming so if you have any additional questions along the way, post a comment on your issue and @mention someone from the @niteoweb/peopleops team. Alternatively, when you are a further down the onboarding issue, you are always welcome to ask questions in the #operations channel on Slack.
+Congratulations on completing the interview process, and joining us here at Niteo. We very much looking forward to having you on the team! This page is designed to help you navigate the onboarding process from reading the Handbook to completing the onboarding checklist.
```

Comment:
`We are very much looking forward` or `We very much look forward`.

Reply:
Good catch :+1: